packages/overlays/src/LocalOverlayController.js

```diff
@@ -113,7 +141,22 @@ export class LocalOverlayController {
     this.isShown ? this.hide() : this.show();
   }
 
-  _createOrUpdateOverlay(shown = this._prevShown, data = this._prevData) {
+  // Popper does not export a nice method to update an existing instance with a new config. Therefore we recreate the instance.
+  // TODO: Send a merge request to Popper to abstract their logic in the constructor to an exposed method which takes in the user config.
+  updatePlacementConfig(config = this.placementConfig) {
+    this.placementConfig = {
+      ...this.placementConfig,
+      ...config,
```

Comment:
this will override all modifiers right?

how about moving the logic from the constructor to here and then call this method in the constructor?

if needed there could be a flag to not do the create/destroy?

Reply:
Done, but it always recreates popper instance upon updating config now. Whether the user sees the difference depends on whether it is shown or not. I did not see a use case where you want to update the placement config but not update the popper instance with it.